grafanalib/core.py

```diff
@@ -335,7 +336,7 @@ class CloudWatchTarget(object):
     highResolution = attr.ib(default=False, validator=instance_of(bool))
 
     refId = attr.ib(default="")
-    datasource = attr.ib(default="")
+    dataSource = attr.ib(default="")
```

Comment:
`data(sS)ource` is super inconsistent all over in grafanalib core - can we punt on this for now?